### DISCUSSION TITLE
backup procedure

### DISCUSSION THREAD

**Discussion by on1arf** [goauthentik/authentik#8077] [Q&A]
Hi all,

Perhaps a stupid question, but I did not find the informaiton in the documentation.

What is the best procedure to backup authentik (dockerised version)?
Just backup the postgres database and make a copy of the docker-compose file? Anything else?
Can you just make the backup while authentik is running, or do you have to put it in some special 'maintenance mode?' or so?

How crucial is the dump.rdb file in the redis container?

Thanks! :-)

Kr.

**Comment by nogweii** [comment]
There used to be a built-in backup method, but now the strategy is two things that I'm aware of:

- Backup the database
- Back up any volumes/files. (For example, blueprints or images that have been uploaded.)
